x/wasm/ioutils/ioutil.go

```diff
@@ -11,7 +11,21 @@ import (
 )
 
 // Uncompress expects a valid gzip source to unpack or fails. See IsGzip
-func Uncompress(gzipSrc []byte, limit uint64) ([]byte, error) {
+func Uncompress(gzipSrc []byte) ([]byte, error) {
```

Comment:
No need to copy the code. Instead of adding a new method, you can simply pass `math.MaxUint` as limit. This makes it also more readable on the caller, IMHO.